src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -107,7 +107,17 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
 
         [Option(
             "baseline",
-            HelpText = "A Sarif file to be used as baseline")]
+            HelpText = "A SARIF file to be used as baseline.")]
         public string BaselineSarifFile { get; set; }
+
+        [Option(
+            "automationId",
```

Comment:
>automationId [](http://example.com/codeflow?start=13&length=12)

why aren't these in common options base? won't we need them for other commands? i guess we can move them later if so.

Reply:
just moved! :)